### PR TITLE
add dompurify v0.7.2

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/cure53/DOMPurify
**Watch 47
  Star 620
 Fork 47**
@LeaYeh could you help me check out the PR for #6138  thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/dompurify/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "mathml",
+    "security",
+    "secure",
+    "sanitizer",
+    "sanitize",
+    "filter",
+    "purify"
+  ],
+  "author": "Mario Heiderich <mario@cure53.de> (https://cure53.de/)",
+  "license": [
```
Comment (LeaYeh) [inline ajax/libs/dompurify/package.json]: can we have two licence in same time?

Comment (maruilian11) [inline ajax/libs/dompurify/package.json]: @LeaYeh yes, we added lib with two license before.
@PeterDaveHello do we must modify `license` to `licenses`?

Comment (PeterDaveHello) [inline ajax/libs/dompurify/package.json]: Should use licenses as is an array.

Comment (maruilian11) [inline ajax/libs/dompurify/package.json]: ok.

**Comment by LeaYeh** [comment]
ok got that. 
could u rebase it?

**Comment by maruilian11** [comment]
@LeaYeh i have rebased it ~

**Comment by LeaYeh** [comment]
@PeterDaveHello could you pls check this pr ?

**Comment by PeterDaveHello** [comment]
As we discussed before, I don't merge intern team's PR who's commit parent is two days ago, should within only one day, please rebase with latest master branch.

**Comment by maruilian11** [comment]
@LeaYeh i have rebased it :)

**Comment by LeaYeh** [comment]
@maruilian11 hey, your online test not passed.
@PeterDaveHello sorry for last time about rebase timing, but was last time commit not just one day delay?
Let's try again and rebase it.

**Comment by PeterDaveHello** [comment]
@LeaYeh No it was 2.

**Comment by maruilian11** [comment]
@PeterDaveHello please take a look.

**Pull request merged**